xbmc/input/WindowTranslator.cpp

```diff
@@ -231,6 +234,8 @@ std::string CWindowTranslator::TranslateWindow(int windowId)
 {
   static auto reverseWindowMapping = CreateWindowMappingByID();
 
+  windowId = CWindowTranslator::GetSpecialWindow(windowId);
```

Comment:
`CWindowTranslator::` prefix isn't needed I guess?

Reply:
will remove it thx